UnityProject/Assets/Scripts/Managers/ConnectedPlayer.cs

```diff
@@ -38,11 +38,15 @@ public GameObject GameObject
 		set
 		{
 			gameObject = value;
+			if (Script)
```

Comment:
use ``{`` and ``}`` to comply with the if statement code style guidelines.

Reply:
whats the guideline on 1 line if statements?